CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# 8.0.0
+
+* Switch from JS custom variables to HTML meta tags
+
+  Slimmer now appends page metadata as meta tags instead of setting Google
+  custom variables within a script tag. The Google-specific implementation
+  details have been removed. Any apps that need to report analytics events
+  will require additional Javascript that reads the meta tags and sends the relevant
+  data to the analytics platform.
```

Comment:
Worth cross linking this to the specific JS/docs required for this switch? Good for someone (less familiar with the change) having to upgrade slimmer.